### PR TITLE
Fix/cc 0027 adjust the court position

### PR DESCRIPTION
**Pull request by Beryl-Zhou**

### CODE REVIEW

**Review by jennawan — APPROVED** [review]
LGTM

**Comment by EmilyGong111** [comment]
Screen Shot 2022-06-18 at 4 04 09 pm
Good job! Just remind the Text import is still there. May need to delete it, since it is not used anymore.

File: src/components/ProFullCourt/index.tsx [hunk 1]
```diff
@@ -11,72 +11,64 @@ import BorderDimensionLine from "../BasketballCourt/BorderDimensionLine";
...
-              </Layer>
-            </Provider>
-          </Stage>
-        )}
-      </ReactReduxContext.Consumer>
-
-      {/* TODO: Only for test */}
-      <Check />
-    </Flex>
+    <>
```
Comment (tatoMa) [inline src/components/ProFullCourt/index.tsx:19]: Why need a fragment tag wrapped with the single Flex tag?

File: src/components/ProFullCourt/index.tsx [hunk 2]
```diff
@@ -11,72 +11,64 @@ import BorderDimensionLine from "../BasketballCourt/BorderDimensionLine";
...
+      <Flex
+        position="fixed"
+        top="122px"
```
Comment (tatoMa) [inline src/components/ProFullCourt/index.tsx:22]: Maybe adjust this property to make the whole canvas a little lifted up and give the whole canvas more space on the screen?

Comment (Beryl-Zhou) [reply src/components/ProFullCourt/index.tsx:22]: got it~~~

**Review by EmilyGong111 — APPROVED** [review]

File: src/components/ProFullCourt/index.tsx (hunk 1 above)
Comment (Beryl-Zhou) [reply src/components/ProFullCourt/index.tsx]: got it~~~ already fixed

File: src/components/ProFullCourt/index.tsx (hunk 2 above)
Comment (Beryl-Zhou) [reply src/components/ProFullCourt/index.tsx]: fixed~~Cheers~~~~~~~

**Pull request merged**